website/content/docs/nia/cli/start.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+page_title: Start Command
+description: >-
+  Consul-Terraform-Sync supports start command for starting CTS as a daemon
+---
+
+# start
+
+The `start` command starts Consul-Terraform-Sync (CTS) as a daemon. When running as a daemon, there is no default configuration to start CTS in a meaningful way. Setting a configuration flag `-config-file` or `-config-dir` is required. For example:
+
+```shell-session
+$ consul-terraform-sync start -config-file=config.hcl
+```
+
+To view a list of available flags, use the `-help` or `-h` flag.
+
+### Modes
+
+CTS can be run as a daemon in different modes.
+
+#### Long-running Mode
+
+Flag: none
+
+Behavior: This is the default mode in which CTS passes through a once-mode phase and then turns into a long-running process. During the once-mode phase, the daemon will exit with a non-zero status if it encounters an error. After successfully passing through once-mode phase, it will begin a long-running process in which errors are logged and exiting is not expected behavior. Once beginning the long-running process, the daemon serves any API and command requests.
```

Comment:
Nit, might be just me, but the highlighted sentence feels a bit convoluted. I am thinking something like this would be simpler?
When the long-running process begins, CTS daemon starts serving API and command requests.
I think because that "Once ..." in the beginning of the sentence is confusing me with the 'once-mode' references in the same paragraph.

Reply:
Sentence referred to: `Once beginning the long-running process, the daemon serves any API and command requests.`